package.json

```diff
@@ -40,7 +40,8 @@
     "react-dom": "19.0.0-rc-66855b96-20241106",
     "react-error-boundary": "^4.1.2",
     "tailwind-merge": "^2.5.4",
-    "tailwindcss-animate": "^1.0.7"
+    "tailwindcss-animate": "^1.0.7",
+    "ws": "^8.18.0"
```

Comment:
Let's remove the `tailwindcss-animate` and `ws` deps. We try to keep the dependencies of these templates to a minimum.